log/log.go

```diff
@@ -33,8 +36,8 @@ func configureHumanReadable() {
 // Data contains structured log data
 type Data map[string]interface{}
 
-// Context returns a context ID from a request (using X-Request-Id)
-func Context(req *http.Request) string {
+// GetRequestID returns the request ID from a request (using X-Request-Id)
+func GetRequestID(req *http.Request) string {
 	return req.Header.Get("X-Request-Id")
```

Comment:
It would probably better if "X-Request-Id" was a constant in the `common` package, since it is used across multiple packages